### PR TITLE
Add support for handling anonyimze some pii data

### PR DESCRIPTION
**Pull request by sunkan**
Sometime it's nice to log data when an app is staging and development but that data shouldn't be added in it's full form in the production app.

This is what the pull-request tries to solve by removing or anonymizing some data.

### CODE REVIEW

File: src/PII/Anonymizer/PasswordAnonymizer.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	}
+
+	public function addField(string $field): void
+	{
+		$this->fields[] = $field;
+	}
+
+	public function support(string $key): bool
+	{
+		return in_array($key, $this->fields, true);
```
Comment (bix0r) [inline src/PII/Anonymizer/PasswordAnonymizer.php]: @sunkan Don't you want to just use `self::PASSWORD` for if key contains this? I doubt we find too many cases where someone adds `_password` as the key.

Comment (sunkan) [reply src/PII/Anonymizer/PasswordAnonymizer.php]: @bix0r check out line `10` and `19`

or am I miss understanding the problem?

Comment (bix0r) [reply src/PII/Anonymizer/PasswordAnonymizer.php]: @sunkan  I don't see how that could catch a field called `user_password`. I guess you should have something like
```
public function support(string $key): bool {
  return in_array($key, $this->fields, true) || strpos($key, self::_PASSWD) !== false;
}
```
Maybe add `user_password` as to the tests?

File: src/PII/Processor.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
+<?php declare(strict_types=1);
+
+namespace Stefna\Logger\PII;
+
+use Stefna\Logger\PII\Anonymizer\Anonymizer;
+use Stefna\Logger\PII\Anonymizer\CardAnonymizer;
+use Stefna\Logger\PII\Anonymizer\PasswordAnonymizer;
```
Comment (bix0r) [inline src/PII/Processor.php]: @sunkan Remove this I guess

**Review by bix0r — APPROVED** [review]
LGTM

**Pull request merged**